cmd/nvidia-device-plugin/main.go

```diff
@@ -149,6 +149,11 @@ func main() {
 			Usage:   "the strategy to use to discover devices: 'auto', 'nvml', or 'tegra'",
 			EnvVars: []string{"DEVICE_DISCOVERY_STRATEGY"},
 		},
+		&cli.IntSliceFlag{
+			Name:    "imex-channel-id",
```

Comment:
```suggestion
			Name:    "imex-channel-ids",
```

Reply:
I always wonder about this. This does probably make it clearer that it's a repeatable field.

Reply:
I see -- because when you pass it its not `--imex-channel-ids=0,1`, its `--imex-channel-ids=0 --imex-channel-ids=1`...

My argument was more for consistence with the envvar of `IMEX_CHANNEL_IDS`. It felt weird to me to have the flag called `imex-channel-id` but then the envvar `IMEX_CHANNEL_IDS`.

Reply:
To be clear, the following are equivalent:
```
--imex-channel-ids=0,1
```
and:
```
--imex-channel-ids=0 --imex-channel-ids=1
```